### PR TITLE
Onboarding active campaign docs

### PR DESCRIPTION
**Pull request by utsabc**
## Description of the change

Documentation for Active Campaign destination

### CODE REVIEW

File: destinations/active-campaign.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+Once you have confirmed that the platform supports sending events to ActiveCampaign, perform the steps below:
+
+* From your [RudderStack dashboard](https://app.rudderlabs.com/), add the source and ActiveCampaign as a destination.
+
+{% hint style="info" %}
+Please follow our guide on [How to Add a Source and Destination in RudderStack](https://docs.rudderstack.com/how-to-guides/adding-source-and-destination-rudderstack) to add a source and destination in RudderStack.
+{% endhint %}
+
+{% hint style="warning" %}
+Do not add native Adjust SDK to your project as it will prevent you from successful integration. Please refer to the screenshot below for more details:
```
Comment (ameypv-rudder) [inline destinations/active-campaign.md:34]: Please review this - I think it is not relevant for the ActiveCampaign destination.

cc @arnab-p @utsabc

File: destinations/active-campaign.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+```
+{% hint style="info" %}
+The `tags` property should contain an array of tags which you want to associate with the user. If any tag is already created in the destination previously, RudderStack will automatically skip the creation of that tag.
+{% endhint %}
+
+### Custom fields
+
+ActiveCampaign also supports updating a contact’s custom fields with this RudderStack integration. 
+
+To send custom fields to ActiveCampaign, you will need to first create the custom fields in ActiveCampaign for each custom field that you want to send. Then when you call `identify` with keys that match those traits, the custom fields for that contact will be updated. You can use the `fieldInfo` trait to set values to the custom fields.
```
Comment (ameypv-rudder) [inline destinations/active-campaign.md:152]: Rewrote this section for more clarity. Please review.

**Review by ameypv-rudder — CHANGES_REQUESTED** [review]

File: destinations/active-campaign.md (hunk 1 above)
Comment (utsabc) [reply destinations/active-campaign.md:34]: Yep, my bad will fix this

File: destinations/active-campaign.md (hunk 2 above)
Comment (utsabc) [reply destinations/active-campaign.md:152]: Looks good

**Review by arnab-p — APPROVED** [review]

**Review by ameypv-rudder — APPROVED** [review]

**Pull request merged**